def get_error(code, message):
    return {'code': code, 'message': message}
